aries_cloudagent/core/event_bus.py

```diff
@@ -14,10 +14,11 @@
 class Event:
     """A simple event object."""
 
-    def __init__(self, topic: str, payload: Any = None):
+    def __init__(self, topic: str, payload: Any = None, endpoint: str = None):
```

Comment:
We should not be modifying the event class. At this point, the base event class should be considered complete. Things like this additional endpoint or other attributes that we want/need on an event should be in the payload.

Reply:
I asked Adam about this here https://github.com/burdettadam/aries-cloudagent-python/commit/22c3b35e0fc7e13efcc81b2c92fa2a27394dce6c#r51981303